### PR TITLE
FOGL-3033 (gui) remove unwanted tab whitspaces from log text; removed hard-coded name

### PR DESCRIPTION
**Pull request by praveen-garg**
of command / action performed and show as received from API

Signed-off-by: Praveen Garg <praveen.garg@nerdapplabs.com>

Patch on https://github.com/foglamp/foglamp-gui/pull/490

### CODE REVIEW

**Comment by praveen-garg** [comment]
unwanted-space-in-logtext-pre

**Review by ashish-jabble — APPROVED** [review]
Looks good as per screenshot

**Review by MonikaSharma06 — APPROVED** [review]

**Review by shariq1501 — APPROVED** [review]

**Comment by ashish-jabble** [comment]
I believe we should not merge this PR until FOGL-3165 core PR merge. Now on GUI name is empty for package logs on fetch available package call

package_log

**Pull request merged**
